src/WebexSDKAdapter.js

```diff
@@ -12,5 +12,23 @@ export default class WebexSDKAdapter {
   constructor(sdk) {
     this.peopleAdapter = new PeopleSDKAdapter(sdk);
     this.roomsAdapter = new RoomsSDKAdapter(sdk);
+    this.sdk = sdk;
+  }
+
+  /**
+   * Call to register the device to the cloud and start the
+   * mercury connection to utilize the internal services
+   */
+  async connect() {
+    await this.sdk.internal.device.register();
+    await this.sdk.internal.mercury.connect();
+  }
+
+  /**
+   * Call to stop the mercury connection and unregister the device from the cloud
```

Comment:
I think we should make this description a little more abstract, in case we do more things with method in the future.
Here's my -more generic- suggestion:
```
Disconnect from Webex services, closing any connections SDK may have opened.